### PR TITLE
Fix template dublication

### PR DESCRIPTION
**Pull request by danielfriis**

### CODE REVIEW

File: app/controllers/template_dublications_controller.rb [hunk 1]
```diff
@@ -1,17 +1,20 @@
 class TemplateDublicationsController < ApplicationController
+  # rubocop:disable Metrics/LineLength
```
Comment (kaspernj) [inline app/controllers/template_dublications_controller.rb]: Image

Comment (danielfriis) [inline app/controllers/template_dublications_controller.rb]: 😂 How would you go about it then?

Comment (kaspernj) [inline app/controllers/template_dublications_controller.rb]:
```ruby
includes = [
  :template_fields,
  {text_template_field_values: :template_field},
  {image_template_field_values: :template_field},
  {url_template_field_values: :template_field}
]

@new_template = @template.deep_clone include: includes, use_dictionary: true do |original, kopy|
  kopy.name = "#{original.name} (#{controller_t("shared.copy_noun")})" if kopy.is_a?(Template)
  kopy.file = original.file if kopy.is_a?(ImageTemplateFieldValue)
end
```

Comment (danielfriis) [inline app/controllers/template_dublications_controller.rb]: ~ Mic drop ~
Nice thanks!

Comment (danielfriis) [inline app/controllers/template_dublications_controller.rb]: Fixed in 12bfae5

**Pull request merged**